configurations/operator/perf-serverless-thrpt-1vcpu.yaml

```
@@ -0,0 +1,17 @@
+email_subject_postfix: 'Scylla-Cloud backend'
+email_recipients: ['valerii.ponomarov@scylladb.com']
```

check the email to:
```suggestion
email_recipients: ['scylla-perf-results@scylladb.com', 'operator@scylladb.com']
```

and it would be nice it would have real email report (we'll address in a different PR)